src/offline/component.js

```diff
@@ -260,28 +260,28 @@ export const Controller = class {
      * @type {number}
      * @export
      */
-    this.minZoom = 0;
+    this.minZoom = undefined;
```

Comment:
```suggestion
    this.minZoom;
```
Should be the same. (I don't know if the typing accept that).

Reply:
By the way that's exported but looks not used in the html